packages/p2p/scripts/extract-string.js

```diff
@@ -5,14 +5,19 @@ const getRegexPattern = () => /(i18n_default_text={?|localize\()\s*(['"])\s*(.*?
 const getStringsFromInput = (input, i18n_marker = getRegexPattern()) => {
     const messages = [];
 
-    while ((result = i18n_marker.exec(input)) !== null) {
-        const extracted = result[3];
-        // Replace escape characters.
-        messages.push(extracted.replace(/\\/g, ''));
+    let continue_loop = true;
+    while (continue_loop) {
```

Comment:
based on the eslint [documentation](https://eslint.org/docs/latest/rules/no-cond-assign) we are not allowed to have assignment within the while statement.